### ISSUE TITLE
when package is updated, re-test packages that depend on it

### ISSUE THREAD

**Issue by notestaff** [conda-forge/conda-forge.github.io#428]
When a package is updated, could you set up conda-forge to automatically re-test packages that depend on it and were not pinned to earlier versions?   Also, maybe scan the defaults package periodically to detect updates of packages on which unpinned conda-forge packages depend, and re-test the affected conda-forge packages?

**Comment by CJ-Wright** [comment]
This could be possible (with a dependency graph which is aware of pinned vs. not pinned) but  it *may* put too much strain on the CI system (eg what if a numpy version came out (or even worse python itself), we may have to run a build for thousands of packages).

**Comment by notestaff** [comment]
Is there a way to mark such CI tasks as "lower priority", to be run when there is idle capacity?

**Comment by jakirkham** [comment]
Nope, CIs don't provide us access to their queue honestly for very good reasons. We could cancel and restart job, but even that creates churn that most CIs will try to rate limit. With Travis CI, we already know there is a rate limit.

**Comment by jakirkham** [comment]
Honestly, @pelson raised issue ( https://github.com/conda-forge/conda-forge.github.io/issues/113 ) about randomly testing packages, which is probably a more pragmatic solution.

**Comment by CJ-Wright** [comment]
To some extent this is mitigated by https://abi-laboratory.pro/ for compiled libraries, although it does nothing for APIs.

**Comment by beckermr** [comment]
bump @wolfv
